docs/snippets/providers/zabbix-snippet-autogenerated.mdx

```diff
@@ -8,12 +8,18 @@ This provider requires authentication.
 - **verify**: Verify SSL certificates (required: False, sensitive: False)
```

Comment:
@becker please note the header of this file, it's auto-generated from https://github.com/keephq/keep/blob/main/keep/providers/zabbix_provider/zabbix_provider.py#L70 . In order to make the test pass, you need to update the python file and trigger 
```
scripts/docs_render_provider_snippets.py
```